tethysapp/water_data_explorer/public/vendor/bootstrap-3.3.6-dist/css/bootstrap.css

```diff
@@ -3808,10 +3808,10 @@ tbody.collapse.in {
 .btn-group-justified > .btn-group .dropdown-menu {
```

Comment:
There are 20 or so files in `tethysapp/water_data_explorer/public/vendor` that have been modified that we should probably avoid modifying if we can. These files are 3rd-party libraries that have been copied into the app's source code.

Don't worry about doing anything about these files for the moment. Get all the other changes/comments addressed done first, and then we can look at what to do with these files.

Reply:
Vendored packages were removed and CDNs were used to get the dependencies instead.